### ISSUE TITLE
Check non-boolean variable in "Feedback value" field

### ISSUE THREAD

**Issue by stiang** [nguyenquyhy/Flight-Tracker-StreamDeck#28]
Is there some way to check for equality against a non-boolean variable in the "Feedback value" field?

For example, while I am waiting for my Honeycomb Bravo (which will give me a proper flaps lever), I have mapped the following toggles in a folder on my Stream Deck:

- FLAPS_UP
- FLAPS_1
- FLAPS_2
- FLAPS_3
- FLAPS_DOWN

These map to the numerical values 0-4 in the `FLAPS HANDLE INDEX` variable. 

So for, say, FLAPS_2, I would like it to show the green "active" indicator when FLAPS HANDLE INDEX == 2, but I can’t figure out a way to put such an expression in the "Feedback value" field.

**Comment by nguyenquyhy** [comment]
Just use ==2 ;)

**Comment by stiang** [comment]
Oh, guess I should have tried the obvious before posting :-P Actually, I believe I tried it with spaces before and after the `==` part, but it looks like whitespace isn’t permitted.

Is this documented somewhere? I spent quite a bit of time researching this, and trying `:2`, `__2` etc. to no avail. Guess I must have missed it if it is.

Anyway, thanks a lot for helping with my particular issue, but also for creating this piece of software - it really makes the Stream Deck quite nice to use with MSFS2020.
